simulator.py

```diff
@@ -108,41 +108,83 @@ def members():
     def log_encounter(human1, human2, location, duration, distance, time):
 
         human1.events.append(
-            {
-                'human_id': human1.name,
-                'time': time,
-                'event_type': Event.encounter,
-                'payload': {
-                    'encounter_human_id': human2.name,
-                    'other_human_is_infected': human2.is_sick,
-                    'other_human_symptoms': human2.symptoms,
-                    'other_human_test_results': human2.test_results,
+            {   'encounter': {
+                    'time': time,
+                    'event_type': Event.encounter,
                     'duration': duration,
                     'distance': distance,
                     'location_type': location.location_type,
                     'contamination_prob': location.cont_prob,
                     'lat': location.lat,
                     'lon': location.lon,
+                    'obs_lat': human1.obs_lat,
+                    'obs_lon': human1.obs_lon,
+                  },
+                'human1': {
+                    'human_id': human1.name,
+                    'age': human1.age,
+                    'carefullness': human1.carefullness,
+                    'is_infected': human1.is_sick,
+                    'infection_timestamp': human1.infection_timestamp,
+                    'infectiousness': human1.infectiousness,
+                    'reported_symptoms': human1.reported_symptoms,
+                    'symptoms': human1.symptoms,
```

Comment:
outside the scope of the PR: I just want to point out that the symptoms are not set at a given moment of time for a human as we're drawing probability in the property directly.